tests/test_exceptions.py

```diff
@@ -0,0 +1,9 @@
+class TestNotCallableError:
```

Comment:
I don't think it makes sense to open a new file (or class for that matter) here. Just put it along with the callable tests.